res/xml/qwertz_cs_minimal.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is encoded as `latin1` but must be UTF-8 to be parsed by android's build tools. I re-encoded it and fixed the escaping of some keys on this branch: https://github.com/Julow/Unexpected-Keyboard/tree/380-reencoded

Reply:
Oops, the encoding is a silly mistake, sorry. The missing escaping is caused by the `custom layout` feature. Adding the layout using the `custom layout` feature requires characters **not** to be escaped. Should I open an issue for this?

Reply:
Interesting ! Yes, an issue would be useful.

Reply:
The issue is opened #381

Reply:
Thanks!

Reply:
Thanks! Do you wish that I push the re-encoding commit to your branch directly ?

Reply:
Yes please :)

Reply:
There was some more encoding problems, so I pushed a second commit. I also made the layout available in the option.